src/components/ExperienceDetail/Article/SectionBlock.js

```diff
@@ -2,12 +2,14 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { P } from 'common/base';
 import styles from './SectionBlock.module.css';
+import OverallRating from 'common/OverallRating';
 
-const SectionBlock = ({ subtitle, content }) => (
+const SectionBlock = ({ subtitle, content, rating }) => (
   <section>
     {subtitle && (
       <P size="l" bold className={styles.heading}>
-        {subtitle}
+        <div>{subtitle}</div>
+        {rating && <OverallRating rating={rating} />}
```

Comment:
Suggestion: `rating` 是數字，若為 `0` 則會顯示出來。建議用 `?:`

```suggestion
        {rating ? <OverallRating rating={rating} /> : null}
```